roles/content/tasks/3_repo_sync.yml

```diff
@@ -0,0 +1,34 @@
+- name: repo sync | get all repositories
+  oasis_roles.satellite.resource_info:
+    username: "{{ satellite_admin_username }}"
+    password: "{{ satellite_admin_password }}"
+    server_url: "{{ satellite_server_url }}"
+    resource: repositories
+  register: repositories
+
+- name: repo sync | Kick off repository sync tasks
+  oasis_roles.satellite.repository_sync:
+    username: "{{ satellite_admin_username }}"
+    password: "{{ satellite_admin_password }}"
+    server_url: "{{ satellite_server_url }}"
+    product: "{{ item.product.name }}"
+    repository: "{{ item.name }}"
+    organization: "{{ satellite_organization }}"
+  loop: "{{ repositories.resources }}"
+  when: item.url  # Not all repositories have a URL
+  async: 999999
```

Comment:
fwiw, we recently removed that example from the docs, as it resulted in OOMed systems in a few cases (too many parallel syncs)

Reply:
removed from code@evgeni